chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/svap/CreateActionPermissions.java

```diff
@@ -0,0 +1,22 @@
+package gov.healthit.chpl.permissions.domains.svap;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.permissions.domains.ActionPermissions;
+
+@Component("svapCreateActionPermissions")
+public class CreateActionPermissions extends ActionPermissions {
+
+    @Override
+    public boolean hasAccess() {
+        return  getResourcePermissions().isUserRoleAdmin()
+            || getResourcePermissions().isUserRoleOnc()
+            || getResourcePermissions().isUserRoleOncStaff();
```

Comment:
Are we letting ONC_STAFF do this? I saw admin and onc in the ticket and I thought we were only letting ONC_STAFF deal with scheduled jobs for now.